ar-setup.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+artifact_registry_location=us-central1
+project=$(gcloud config get-value core/project)
+readonly project_var_name_gcp_ar_repo=GCP_AR_REPO
+readonly project_var_name_gcp_ar_key=GCP_AR_KEY
+
+usage () {
+    echo "usage: ./ar-setup.sh --app-name <app_name> --gitlab-access-token <token> --app-config-repo <app-config-repo> [--project <gcp_project>] [--artifact-registry-location <location>]"
+	echo "  OR   ./ar-setup.sh --app-name=<app_name> --gitlab-access-token=<token> --app-config-repo=<app-config-repo> [--project=<gcp_project>] [--artifact-registry-location=<location>]"
```

Comment:
Keeping one is enough. I'd prefer the latter one (--option=xxx).

Reply:
Done. Thanks for the suggestion.